examples/FW-APSP/FloydRecursiveSerialKernelDF.h

```diff
@@ -0,0 +1,204 @@
+#ifndef FLOYD_RECURSIVE_SERIAL_KERNEL
```

Comment:
We should not maintain this file as a copy of the OMPI version. Instead we should recompile the same file twice, once with PML support and once without.

Reply:
I guess this comment targeted a file in Open MPI?

Reply:
That's an automatic corrector mistake. It should not have been OMPI, but OMP (like Open MP)